linux_os/guide/system/auditing/auditd_configure_rules/audit_time_rules/audit_rules_time_stime/ansible/shared.yml

```diff
@@ -0,0 +1,8 @@
+# platform = multi_platform_all
+# reboot = false
+# strategy = restrict
+# complexity = low
+# disruption = low
+
+{{{ ansible_audit_augenrules_add_syscall_rule(syscalls=["stime"], key="audit_time_rules") }}}
```

Comment:
This will need to be rewritten because we want only 32 bit syscall. Stime is not in the table of 64 bit syscalls for Audit.

Reply:
Nice catch, I'll rework it based on the macro update.

Reply:
Rebased and reworked based on macro with independent arch support